packages/server/src/routes/index.ts

```diff
@@ -1,7 +0,0 @@
-import express from 'express'
-import WellKnown from './well-known'
-const router = express.Router()
-
-router.use('/.well-known', WellKnown)
-
-export default router
```

Comment:
this router wasn't even getting hooked up anywhere

also this is a did:web route so it doesn't really make sense to server did:plc off of it